doc/api/events.md

```diff
@@ -167,6 +167,9 @@ myEmitter.emit('error', new Error('whoops!'));
 ```
 
 ## Class: EventEmitter
+<!-- YAML
+added: v0.1.90
```

Comment:
Sorry, it's actually [v0.1.26](https://github.com/nodejs/node/commit/70293a43c9d4075165e2f7f623fa86339eb9d27f).